ajax/libs/xterm/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "xterm",
+  "filename": "xterm.min.js",
+  "version": "2.3.2",
+  "description": "Full xterm terminal, in your browser",
+  "homepage": "https://xtermjs.org/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sourcelair/xterm.js"
+  },
+  "keywords": [
+    "xterm",
+    "terminal",
+    "browser-tty"
```

Comment:
Is it a type error here?
If yes, please modify it and also add some other keywords to let users be able to find the library in the first page of the searching result in a search engine like Google. E.g `JavaScript` and `browser`.